azure/resource-group/variables.tf

```diff
@@ -43,8 +43,8 @@ variable "environment" {
   type        = string
 
   validation {
-    condition     = contains(["Dev", "Prod", "Staging", "Test"], var.environment)
-    error_message = "The 'environment' value is invalid. Valid options are 'Dev', 'Prod','Staging', 'Test'."
+    condition     = contains(["Dev", "Kitchen", "Production", "Staging", "Test"], var.environment)
```

Comment:
I believe it would be better to user short name for the environments when possible, like: prod, stag, since they will be used on resource naming as well where there is a chars limitation.

Reply:
I understand your point of view, especially since it may cause us problems in the future, but I'm trying to stick with Nmbrs' tagging strategy. Shouldn't we follow it?

Reply:
Since our last conversation with developers, I understand that they want us to stick with it, and as you mentioned, it will be difficult to give accurate names for modules with naming restrictions, such as keyvaults, storage accounts, or even DNS zones. 

Also, for those modules, we need to implement methods for trimming the naming anyway, because it takes a lot more effort to predict, calculate and limit the length of all possible combinations that use the caf prefix, 'product' variable, 'environment' variable, and random sequence.

Reply:
I got it @filipefigueredo but then you must drop the production one since the Nmbrs convention says it doesnt need to be exposed.